websites/I/India Today News/presence.ts

```diff
@@ -0,0 +1,541 @@
+		)?.textContent;
+		presenceData.state = storyDate;
+		presenceData.smallImageKey = video.paused ? Assets.Pause : Assets.Play;
+		presenceData.smallImageText = video.paused ? strings.pause : strings.play;
+		presenceData.buttons = [
+			{
+				label: "Watch video",
+				url: href,
+			},
+		];
+
+		if (showTimestamp) {
+			presenceData.startTimestamp = startTimestamp;
+			presenceData.endTimestamp = endTimestamp;
+		}
+
+		if (video.paused) delete presenceData.endTimestamp;
+
+		if (pathname.includes("s") && !pathname.includes("/video/")) {
+			presenceData.details = `Viewing ${pathname
+				.split("/videos")[0]
+				?.replace("/", "")
+				?.split("/")[1]
+				.toUpperCase()} Videos`;
+			presenceData.smallImageKey = Assets.Reading;
+			presenceData.smallImageText = "India Today News - Videos";
+			presenceData.buttons = [
+				{
+					label: "View Videos",
+					url: href,
+				},
+			];
+		}
+	} else if (pathname.includes("/weather/")) {
+		presenceData.details = document
+			.querySelector("div.wtr_ttl_lhs")
+			?.querySelector("h1")
+			?.textContent.split("Today")[0];
+		presenceData.state = document
+			.querySelector("div.wtr_ttl_lhs")
+			?.querySelector("h1")
+			?.textContent.split("Today")[1];
+		presenceData.smallImageKey = document
+			.querySelector("span.wkl_li_icn")
+			?.querySelector("img")?.src;
+		presenceData.smallImageText = `${
+			document.querySelector("span.wtr_tmp_txt")?.textContent
+		} - ${
+			document.querySelector("div.wtr_tmp_rhs")?.querySelector("strong")
+				?.textContent
+		}°C`;
+		presenceData.buttons = [
+			{
+				label: "View Weather",
+				url: href,
+			},
+		];
+	} else if (pathname.includes("/story")) {
+		presenceData.details = document.querySelector(
+			"h1.Story_strytitle__MYXmR"
+		)?.textContent;
+		presenceData.state = storyDate;
+		presenceData.largeImageKey = document
+			.querySelector("div.Story_associate__image__bYOH_")
+			?.querySelector("img")?.src;
+		presenceData.smallImageKey = document
+			.querySelector("a.jsx-99cc083358cc2e2d")
+			?.querySelector("img")?.src;
+		presenceData.smallImageText = authorTitle;
+		presenceData.buttons = [
+			{
+				label: "Read Article",
+				url: href,
+			},
+		];
+
+		if (pathname.includes("/india/story"))
+			presenceData.largeImageKey = Assets.Logo;
+
+		if (multipleAuthors) {
+			presenceData.smallImageKey = Assets.Logo;
+			presenceData.smallImageText = "Multiple Authors";
+		}
+	} else if (video && !isNaN(video.duration) && pathname.includes("livetv")) {
+		presenceData.details = "Watching Live TV";
+		presenceData.state = document
+			.querySelector("h1.Livetv_titlewithicon__Z2jnL")
+			?.textContent.toUpperCase()
+			?.replace("LIVE TV", "");
+		presenceData.smallImageKey = video.paused ? Assets.Pause : Assets.Live;
+		presenceData.smallImageText = video.paused ? strings.pause : "Live";
+		presenceData.buttons = [
+			{
+				label: "Watch Live TV",
+				url: href,
+			},
+		];
+	}
+
+	if (!showButtons) delete presenceData.buttons;
```

Comment:
Otherwise you'll delete it for no reason.
```suggestion
	if (!showButtons && presenceData.buttons) delete presenceData.buttons;
```

Reply:
The code rn:
if buttons are disabled delete buttons
My suggested change:
If buttons are disabled and THERE are buttons then delete buttons

Reply:
ye